### PR TITLE
Updated resident's page UI based on the new designs

### PR DESCRIPTION
**Pull request by bogdan-zaharia**
Link to Jira card: https://hackney.atlassian.net/browse/DOC-301

Updated resident's page UI based on the new designs

### CODE REVIEW

File: src/pages/teams/[teamId]/dashboard/residents/[residentId]/index.tsx [hunk 1]
```diff
@@ -47,8 +48,18 @@ const ResidentPage: NextPage<WithUser<ResidentPageProps>> = ({
...
+      <div
+        className="toReview govuk-form-group--error"
+        style={{
+          borderLeftColor: '#F0D232',
+          backgroundColor: '#FFFBF4',
+          paddingTop: '1.5em',
+        }}
+      >
+        <h2 className="lbh-heading-h3">
+          <SVGSymbol status={'toReview'} />
```
Comment (chrishutchinson) [inline src/pages/teams/[teamId]/dashboard/residents/[residentId]/index.tsx:60]:
```suggestion
          <SVGSymbol status="toReview" />
```

File: src/pages/teams/[teamId]/dashboard/residents/[residentId]/index.tsx [hunk 2]
```diff
@@ -73,8 +84,11 @@ const ResidentPage: NextPage<WithUser<ResidentPageProps>> = ({
         </EvidenceList>
       </div>
 
-      <div className="reviewed">
-        <h2 className="lbh-heading-h3">Reviewed</h2>
+      <div className="reviewed evidence-list">
+        <h2 className="lbh-heading-h3">
+          <SVGSymbol status={'reviewed'} />
```
Comment (chrishutchinson) [inline src/pages/teams/[teamId]/dashboard/residents/[residentId]/index.tsx:89]:
```suggestion
          <SVGSymbol status="reviewed" />
```

File: src/pages/teams/[teamId]/dashboard/residents/[residentId]/index.tsx [hunk 3]
```diff
@@ -99,8 +113,11 @@ const ResidentPage: NextPage<WithUser<ResidentPageProps>> = ({
       </div>
 
       {rejectedDocumentSubmissions && rejectedDocumentSubmissions.length > 0 && (
-        <div className="rejected">
-          <h2 className="lbh-heading-h3">Rejected</h2>
+        <div className="rejected evidence-list">
+          <h2 className="lbh-heading-h3">
+            <SVGSymbol status={'rejected'} />
```
Comment (chrishutchinson) [inline src/pages/teams/[teamId]/dashboard/residents/[residentId]/index.tsx:118]:
```suggestion
            <SVGSymbol status="rejected" />
```

**Review by chrishutchinson — APPROVED** [review]

File: src/pages/teams/[teamId]/dashboard/residents/[residentId]/index.tsx (hunk 1 above)
Comment (bogdan-zaharia) [reply src/pages/teams/[teamId]/dashboard/residents/[residentId]/index.tsx]: addressed

File: src/pages/teams/[teamId]/dashboard/residents/[residentId]/index.tsx (hunk 2 above)
Comment (bogdan-zaharia) [reply src/pages/teams/[teamId]/dashboard/residents/[residentId]/index.tsx]: addressed

File: src/pages/teams/[teamId]/dashboard/residents/[residentId]/index.tsx (hunk 3 above)
Comment (bogdan-zaharia) [reply src/pages/teams/[teamId]/dashboard/residents/[residentId]/index.tsx]: addressed

**Review by chrishutchinson — APPROVED** [review]

**Pull request merged**